### PR TITLE
gh-1414 min amount for delegating to DACs

### PR DESCRIPTION
**Pull request by arbreton**

### CODE REVIEW

File: src/components/DonateButton.jsx [hunk 1]
```diff
@@ -184,7 +184,10 @@ class DonateButton extends React.Component {
 
     let { maxDonationAmount } = this.props;
     if (maxDonationAmount) {
-      if (dacId !== undefined && dacId !== 0) {
+      const amountDAC = parseFloat(maxDonationAmount - maxDonationAmount / 1.03)
```
Comment (aminlatifi) [inline src/components/DonateButton.jsx:187]: If you want to get the amount will be donated to dac, can simply write `maxDonationAmount * 0.03`

**Review by aminlatifi — CHANGES_REQUESTED** [review]
The implementation doesn't address issue #1414 requirement. You should obtain the usd value of original donation and compare it with 40$. If it's less than that ignore donating to DAC

**Comment by arbreton** [comment]
Will check on the USD value

**Comment by aminlatifi** [comment]
We are implementing an api on feathers for hourly rates, you can use it after is merged
https://github.com/Giveth/feathers-giveth/pull/287

**Comment by arbreton** [comment]
Sounds great, will do

**Pull request closed without merging**